.github/workflows/update-prerun-script.yml

```diff
@@ -0,0 +1,93 @@
+name: Update pre build script on Saucelabs
+
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+#  schedule:
+#    - cron: '0 0 1 */2 *'
+permissions:
+  contents: read
+  pull-requests: write
+
+env:
+  SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+  SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+
+jobs:
+  get-previous-version:
+    name: Get Previous Major Version
+    runs-on: ubuntu-latest
+    outputs:
+      previous_version_number: ${{ steps.get-version.outputs.version-number }}
+    steps:
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get Version
+        id: get-version
+        run: |
+          current_version=$(.github/script/get-current-version)
+          echo "::set-output name=version-number::$((${current_version%%.*} - 1))"
```

Comment:
why cant you just add this to the job below? Why do you need a second job?

Reply:
Agree with Michael. This can be organized better. Seem like you need 3 jobs. First one is to download the scripts and upload to SL. Then you can have another 2 jobs to update it in current and prev_version branch.

Reply:
I would combine "get-previous-version" and "update-pre-run-script"

Reply:
Done